src/app/api/resident-request/route.ts

```diff
@@ -0,0 +1,42 @@
+import { NextRequest, NextResponse } from "next/server";
+import { prisma } from "lib/prisma";
+import { RequestStatus } from "@prisma/client"; // Import RequestStatus enumD
+
+export async function GET(request: NextRequest) {
```

Comment:
Don't forget to protect this route and make sure only admins can access it - since it's returning all requests of some type of status. You can import the `auth` function from `auth.ts` and wrap the route with it. I do something similar in the submission form PR